dex/src/main/scala/com/wavesplatform/dex/AddressActor.scala

```diff
@@ -564,9 +565,4 @@ object AddressActor {
   private case class PendingCommand(command: OneOrderCommand, client: ActorRef)
 
   private case class InsufficientBalanceOrder(order: Order, insufficientAmount: Long, assetId: Asset)
-
-  final case class Settings(wsMessagesInterval: FiniteDuration, batchCancelTimeout: FiniteDuration, maxActiveOrders: Int)
```

Comment:
Why? Each component has own settings. It's easy to find the settings in a companion object, because it is used only in this class and only in constructor. Also it is a small case class in most cases.

Reply:
Lets see another point:
Everything we got from application.conf and that is passed through MatcherSettings has it's own dedicated settings

Reply:
MatcherSettings is broken. It *has* all settings, but should only *aggregate* them.

Reply:
Done